src/web/jdv/server/jaialog_store.py

```diff
@@ -107,6 +88,37 @@ def get_title_from_path(path):
 UTIME_PATH = 'goby::health::report/goby.middleware.protobuf.VehicleHealth/_utime_'
 
 
+@dataclass
+@dataclass_json
+class LogDescription:
+    '''Metadata pertaining to a log'''
+
+    bot: str
+    fleet: str
+    
+    filename: str
+    '''File stem for this log (without path, .goby or .h5 extension)'''
+
+    timestamp: float
+    '''UNIX timestamp of the date (from the filename)'''
+
+    duration: Optional[float] = None
+    '''Log duration (in microseconds).  Only present for a log that has been converted to HDF5 format.'''
+
+    size: int = 0
+    '''Log file size (in bytes)'''
+
+
+@dataclass
+@dataclass_json
+class LogDirectory:
+    """A list of available logs with their metadata, and the available space on the storage device"""
```

Comment:
Should we use single or double quotes for the docstrings? I think either is fine, we just need to be more consistent

Reply:
It always comes down to trade-offs haha!